release-notes.hbs.md

```diff
@@ -152,9 +152,27 @@ enable TUF:
 
 - SCST - Store returns an expired certificate error message when a CA certificate expires before the app certificate. For more information, see [CA Cert expires](scst-store/troubleshooting.hbs.md#ca-cert-expires).
 
+
+#### <a id='1-9-0-ssc-ui-ki'></a> v1.9.0 Known issues: Supply Chain UI
+
+- When accessing the supply chain tab in the Tanzu Developer Portal, users might encounter an error related to data.packaging.carvel.dev. The error message displayed is related to permission issues and JSON parsing errors, specifically mentioning that the user "system:serviceaccount:tap-gui:tap-gui-viewer" cannot list resource "packages" in the API group "data.packaging.carvel.dev" at the cluster scope. Additionally, an unexpected non-whitespace character is reported after JSON at position 4.
+
+Workaround: A temporary solution involves applying an RBAC configuration that includes permissions (get, watch, list) for the resources within the data.packaging.carvel.dev API group. This configuration mitigates the issue but it is highlighted that such a requirement should not be mandated for supply chains not generating Carvel packages.
```

Comment:
Does  _**but it is highlighted that such a requirement should not be mandated for supply chains not generating Carvel packages**_ mean
This workaround should not be mandated for supply chains that do not generate Carvel packages.

or even
This workaround is only required for supply chains that generate Carvel packages.

Reply:
For removing the error it is required for all type of supply chains, even if they generate or not Carvel Packages. But that statement refers tois a know issue requesting that permision even when a supply chain does not generate Carvel packages.

Reply:
Maybe

> Workaround: To mitigate against this problem, create an RBAC configuration that includes the `get`, `watch`, and `list` permissions for all resources within the `data.packaging.carvel.dev` API group. These permissions are overbroad — only supply chains that create Carvel packages really need this access — but granting them eliminates the error.